### ISSUE TITLE
More database types (PostgreSQL, etc...)

### ISSUE THREAD

**Issue by tookender** [pterodactyl/panel#4397]
### Is there an existing feature request for this?

- [X] I have searched the existing issues before opening this feature request.

### Describe the feature you would like to see.

I would like to be able to add more database types to assign to servers such as PostgreSQL, SQLite and possibly more.
There is no reason to ever use MySQL over PostgreSQL; MySQL has no transactional DDL, lots of security issues, is owned by Oracle, is slow, disconnects you randomly, allows insane data, is not ANSI-compliant and possibly more.
The only way you can currently do this is to make another server, go through the pain of setting it up, just for the egg to not work.

### Describe the solution you'd like.

The ability to add more database types to servers such as PostgreSQL, SQLite and possibly more in the future.

### Additional context to this request.

I know this has been suggested before, but it was denied. But since pterodactyl has gotten a new maintainer, I would like their opinion on this.

**Comment by tookender** [comment]
Sorry, I did not mean to add SQLite as you can already do that, since it's just a file database, still, support for PostgreSQL and other databases like that would be cool.

**Comment by trenutoo** [comment]
Duplicate of #2025

**Comment by parkervcp** [comment]
This looks like it's not wanting to move to Postgres, but supporting using it to allocate databases for servers. That and supporting other databases as well.

**Comment by tookender** [comment]
Indeed, I just want the ability to choose between various databases when creating a new database.

**Comment by DaneEveritt** [comment]
In the interest of maintainer sanity and cleaning up an enormous backlog of stale feature requests we have moved to requiring all feature requests be discussed within [Discussions](https://github.com/orgs/pterodactyl/discussions/categories/feature-requests). Please consider re-opening this request there if you believe it is still relevant and worth consideration.

See https://github.com/pterodactyl/panel/issues/5491 for additional context.

Thanks.

<sub>_This comment is in the interest of issue tracker maintenance: it does not imply that maintainers are endorsing or dismissing your request._</sub>

**Comment by itzzjustmateo** [comment]
@DaneEveritt  I've hoped that he made a feature request in discussions after that, but he didn't, but because I need this feature for all my upcomming projects (they all will use postgres, and I want to self host it), I made the discussions https://github.com/orgs/pterodactyl/discussions/5608
